addons/im_livechat/models/res_users.py

```diff
@@ -42,7 +42,7 @@ def SELF_WRITEABLE_FIELDS(self):
 
     @api.depends('res_users_settings_id.livechat_username')
     def _compute_livechat_username(self):
-        for user in self:
+        for user in self.sudo():
```

Comment:
It would be best to have the sudo exactly where we need them and not more, so in this PR just before `res_users_settings_id`.

And we try to document why they are acceptable with a small comment, if you look at `# sudo: ` in the code you should find some good (and some less good) examples.